server/broadlistening/pipeline/steps/hierarchical_aggregation.py

```
@@ -1,6 +1,7 @@
 """Generate a convenient JSON output file."""
 
 import json
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)